integration-tests/src/test/resources/istio/istio-sessmigr-template.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2020, 2021, Oracle and/or its affiliates.
```

Comment:
Change the copyright information --> Copyright (c) 2021, Oracle and/or its affiliates.

Reply:
changed